### PR TITLE
Change tanh args for double mach reflection case

### PR DESCRIPTION
**Pull request by rishit2307**

### CODE REVIEW

File: 2d-double-mach-reflection/double-mach-reflection.ini [hunk 1]
```diff
@@ -80,7 +80,7 @@ basedir = .
 basename = double-mach-reflection-{t:.2f}
 
 [soln-ics]
-phi = ((0.5*tanh(1000000*(x - 0.16666667 - (y/1.7320508))) + 0.5))
+phi = (0.5*tanh(min(pow(10,7)*(x - 0.16666667 - (y/1.7320508)), 40)) + 0.5)
```
Comment (FreddieWitherden) [inline 2d-double-mach-reflection/double-mach-reflection.ini]: Using pow here is not needed; 1e7 will do it.

File: 2d-double-mach-reflection/double-mach-reflection.ini [hunk 2]
```diff
@@ -62,7 +62,7 @@ p = p2
 
 [soln-bcs-top]
 type = sup-in-fa
-phi = ((0.5*tanh(1000000*(x - 0.16666667 - (10*t/0.866025404) - (y/1.7320508))) + 0.5))
+phi = (0.5*tanh(min(pow(10,7)*(x - 0.16666667 - (10*t/0.866025404) -(1/1.7320508)), 40)) + 0.5)
```
Comment (FreddieWitherden) [inline 2d-double-mach-reflection/double-mach-reflection.ini]: Compute -(1/1.7320508) as a scalar.

**Comment by FreddieWitherden** [comment]
Can you verify everything still works as expected?

**Comment by rishit2307** [comment]
Yes, confirmed.

**Pull request merged**